### ISSUE TITLE
apple mail bold font 

### ISSUE THREAD

**Issue by AmeliePoulain92** [foundation/foundation-emails#671]
<img width="1440" alt="bold" src="https://cloud.githubusercontent.com/assets/13813409/21353479/daff31f8-c6ce-11e6-8ec8-9da1e2ed7fc7.png">
My Apple mail text (right pic) looks too bold compared to Gmail.

**Comment by davehoran** [comment]
Are you using web fonts? If so, is the text somehow sized in Apple Mail such that there isn't a good size defined in the loaded font set, so it reverts to something else? Grabbing at straws here.

**Comment by AmeliePoulain92** [comment]
I just used -webkit-font-smoothing: antialiased for the body and html and it works fine for me. Why it is not in reset?

**Comment by cossssmin** [comment]
> Why it is not in reset?

I agree, I think it should be there, but I guess it was either unintentionally overlooked, or it's so specific to Webkit on Mac, that it was considered a client-specific setting that users should add in themselves (frameworks should not include client-specific settings).

For example, here's a test in Apple Mail 9, with the `Helvetica, Arial, sans-serif` font-stack and a 16px font size. For control, I used a normal weight on the body text and a bold weight on the button text.

- Left : without `-webkit-font-smoothing: antialiased`
- Right : with `-webkit-font-smoothing: antialiased`

![image](https://cloud.githubusercontent.com/assets/1656595/21356107/2216e2d4-c6d9-11e6-815d-d16b91c87b43.png)

As you can see, the test on the right not only has sharper-looking body text, but it more clearly indicates the font weight used on the button. The left screen shows no truly distinguishable sign of bolded text, which was clearly not my intention in designing it.

Create a Pull Request and mention this issue, so the maintainers can see clear examples :)

**Comment by joeworkman** [comment]
Foundation for Emails has been superseded by [Inky v2](https://github.com/foundation/inky), a complete rewrite in Rust with improved Outlook support, hybrid output mode, bulletproof buttons, validation, and more. This issue will not be addressed in the legacy codebase.

If this issue persists in v2, please open a new issue at https://github.com/foundation/inky/issues
